### DISCUSSION TITLE
Skywalking-python doc environment variable spelling

### DISCUSSION THREAD

**Discussion by Jedore** [apache/skywalking#10553] [General]
In doc https://skywalking.apache.org/docs/skywalking-python/v1.0.0/en/setup/configuration/#plugin-related-configurations, there are some variables like `SW_AGENT_AGENT_COLLECTOR_BACKEND_SERVICES`, `SW_AGENT_PLUGIN_FASTAPI_COLLECT_HTTP_PARAMS`. But in python code https://github.com/apache/skywalking-python/blob/master/skywalking/config.py, relative variables are `SW_AGENT_COLLECTOR_BACKEND_SERVICES`, `SW_PLUGIN_FASTAPI_COLLECT_HTTP_PARAMS`. So need to correct any one?

**Comment by kezhenxu94** [comment]
Hi thanks. But this is fixed in https://github.com/apache/skywalking-python/pull/292

**Reply by wu-sheng** [reply]
I submitted this https://github.com/apache/skywalking-website/pull/584. I think it will fix the docs soon.
